holesky/README.md

```diff
@@ -39,6 +39,10 @@ source .env
 ls $USER_HOME $EIGENLAYER_HOME $EIGENDA_HOME
 ```
 
+:::warning
```

Comment:
can you se `> **_WARNING:_**` ? - this current format you are usinf works for docusaurus but doesn't work for actual readme in github

Reply:
check this line https://github.com/Layr-Labs/eigenda-operator-setup/pull/103/files#diff-3aa5dac1ee35f6b5c5e5a81b0dae3e3918420cbb7cb538e3e3c0e90456854757R100

Reply:
the format I mentioned works for both

Reply:
Good suggestions. I've updated accordingly